packages/pwa-kit-react-sdk/package.json

```diff
@@ -134,8 +135,8 @@
     "node-fetch": "2.6.1",
     "node-html-parser": "^3.3.4",
     "prettier": "1.18.2",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "17.0.2",
```

Comment:
Asking for my understanding, why is this not `^17.0.2`?

Reply:
I don't have a strong opinion, historically we've been pining the version of React. My guess would be that we want to be safe and not get unexpected bugs from new minor releases.. that being said, I do think it's okay to add the caret.

Reply:
I added the caret!